modules/detections_and_time.py

```diff
@@ -37,61 +35,53 @@ def create(cls,
 
         return True, Detection(cls.__create_key, bounds, label, confidence)
 
-    def __init__(self, class_private_create_key, bounds: np.ndarray, label: int, confidence: float):
+    def __init__(
+        self, class_private_create_key: object, bounds: np.ndarray, label: int, confidence: float
+    ) -> None:
         """
         Private constructor, use create() method.
         """
         assert class_private_create_key is Detection.__create_key, "Use create() method"
 
-        # Mixing letters and numbers confuses Pylint
-        # pylint: disable=invalid-name
-        self.x1 = bounds[0]
-        self.y1 = bounds[1]
-        self.x2 = bounds[2]
-        self.y2 = bounds[3]
-        # pylint: enable=invalid-name
+        self.x_1 = bounds[0]
+        self.y_1 = bounds[1]
+        self.x_2 = bounds[2]
+        self.y_2 = bounds[3]
+
         self.label = label
         self.confidence = confidence
 
-    def __repr__(self) -> str:
-        representation = \
-            "cls: " + str(self.label) \
-            + ", conf: " + str(self.confidence) \
-            + ", bounds: " \
-                + str(self.x1) + " " \
-                + str(self.y1) + " " \
-                + str(self.x2) + " " \
-                + str(self.y2)
-
+    def __str__(self) -> str:
+        """
+        To string.
+        """
+        representation = f"cls: {self.label}, conf: {self.confidence}, bounds: {self.x_1} {self.y_1} {self.x_2} {self.y_2}"
```

Comment:
Keep `__str__()` function consistent with other modules. Either just return the string or always assign `representation = f"...."` and return `representation`